torch/quantization/fx/quantize.py

```diff
@@ -669,12 +674,13 @@ def _convert(self, model: GraphModule, is_reference: bool = False,
             model.graph, self.modules, self.patterns,
             custom_module_classes=custom_module_classes)
 
-        quants: Dict[str, Tuple[DefaultQuantizeHandler, Callable]] = \
+        quants: Dict[str, List[Tuple[DefaultQuantizeHandler, Callable]]] = \
             self._find_quants(model.graph, matches)
 
         self.quantized_graph = Graph()
         env: Dict[str, Node] = {}
-        quant_env: Dict[str, Node] = {}
+        # TODO: merge quant_env with env
+        quant_env: Dict[str, Tuple[Node, torch.dtype]] = {}
```

Comment:
can we keep the types of `env` and `quant_env` consistent?  If `quant_env` needs a dtype, can we add the `torch.float` dtype to every entry of `env?

Reply:
yeah, see TODO comment above, I plan to merge them later